### PR TITLE
feat: integrate circleci

### PR DESCRIPTION
**Pull request by ImmanuelSegol**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout:
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@5'
```
Comment (michael1011) [inline .circleci/config.yml]: Why `npm@5`?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          key: dependency-cache-{{ checksum "package.json" }}
+          paths:
+            - ./node_modules
+      - run:
+          name: lint
+          command: npm run lint
+      - run:
+          name: build
+          command: npm run build
+      - store_artifacts:
```
Comment (michael1011) [inline .circleci/config.yml]: We don't have any artifacts to store so this line and everything underneath can be removed

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+          path: test-results.xml
+          prefix: tests
+      - store_artifacts:
+          path: coverage
+          prefix: coverage
+      - store_test_results:
+          path: test-results.xml
```
Comment (michael1011) [inline .circleci/config.yml]: Missing EOF

File: .circleci/config.yml (hunk 1 above)
Comment (ImmanuelSegol) [reply .circleci/config.yml]: Didn't notice will use `lts`

File: .circleci/config.yml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:lts
+    steps:
+      - checkout:
+      - run:
+          name: update-npm
+          command: 'sudo npm install -g npm@lts'
```
Comment (michael1011) [inline .circleci/config.yml]: Isn't NPM installed by default?

Comment (ImmanuelSegol) [reply .circleci/config.yml]: Seem like it is.

**Pull request merged**